app-admin/kubelet-wrapper/files/kubelet-wrapper

```diff
@@ -4,22 +4,60 @@
 # Make sure to set KUBELET_VERSION to an image tag published here:
 # https://quay.io/repository/coreos/hyperkube?tab=tags Alternatively,
 # override $KUBELET_ACI to a custom location.
+#
 
 set -e
 
-if [ -z "${KUBELET_VERSION}" ]; then
-    echo "ERROR: must set KUBELET_VERSION"
+function require_ev_all() {
+    for rev in $@ ; do
+        if [[ -z ${!rev} ]]; then
+            echo ${rev} is not set
+            exit 1
+        fi
+    done
+}
+
+function require_ev_one() {
+    for rev in $@ ; do
+        if [[ ! -z ${!rev} ]]; then
+            return
+        fi
+    done
+    echo One of $@ must be set
     exit 1
+}
+
+if [[ ! -z ${KUBELET_VERSION} ]]; then
+    echo KUBELET_VERSION environment variable is deprecated, please use KUBELET_IMG_TAG instead
+fi
+
+if [[ ! -z ${KUBELET_ACI} ]]; then
+    echo KUBELET_ACI environment variable is deprecated, please use the KUBELET_IMG instead
 fi
 
-KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+KUBELET_IMG_TAG="${KUBELET_IMG_TAG:-${KUBELET_VERSION}}"
+KUBELET_IMG="${KUBELET_ACI}"
+
+require_ev_one KUBELET_IMG KUBELET_IMG_TAG
+
+if [[ ! -z ${KUBELET_IMG_TAG} ]]; then
```

Comment:
Hmm. I'm misinterpreting a lot here. So KUBELET_IMG is actually "repo:tag".

Why are we conflating so many things (img, img_url, tag). Is there a reason we can't just go with something like:

```
KUBELET_REPO
KUBELET_TAG
```

ignore me if this has already been discussed.